stripe_test.go

```diff
@@ -560,25 +546,27 @@ func TestDo_TelemetryEnabled(t *testing.T) {
 		},
 	).(*BackendImplementation)
 
+	type myCreateParams struct {
+		Params `form:"*"`
+		Foo string `form:"foo"`
+	}
+	params := &myCreateParams{
+		Foo: "bar",
+	}
+	params.InternalSetUsage([]string{"llama", "bufo"})
 	for i := 0; i < 2; i++ {
-		request, err := backend.NewRequest(
-			http.MethodGet,
-			"/hello",
-			"sk_test_123",
-			"application/x-www-form-urlencoded",
-			nil,
-		)
-		assert.NoError(t, err)
-
 		var response testServerResponse
-		err = backend.Do(request, nil, &response)
+		err := backend.Call("get", "/hello", "sk_test_xyz", params, &response)
 
 		assert.NoError(t, err)
 		assert.Equal(t, message, response.Message)
 	}
 
 	// We should have seen exactly two requests.
 	assert.Equal(t, 2, requestNum)
+	// The telemetry in the second request should contain the
+	// expected usage
+	assert.Equal(t, telemetry.LastRequestMetrics.Usage, []string{"llama", "bufo"})
```

Comment:
Why not validate next to the RequestDurationMS validation?

Reply:
Fixed!